static/sharing.js

```diff
@@ -88,9 +129,25 @@ function initShareButton(getLink, layout) {
         if (!currentBind) currentBind = $(root.find('.sources a')[0]).data().bind;
 
         function update() {
+            var url = currentBind ? urls[currentBind] || "" : "";
+            if (options.sharingEnabled) {
+                // Is there a better way to get the popup object? There's a field with the element in getLink, but
+                // it's under a jQueryXXXXXX field which I guess changes name for each version?
+                var popoverId = '#' + getLink.attr('aria-describedby');
+                var socialSharing = $(popoverId).find('.socialsharing');
```

Comment:
If anyone knows a better way to get the popup element other than this, please let us know, as this current method feels rather dirty